File: app/src/main/java/com/wyang/study/adapter/SimpleAdapter.java
package com.wyang.study.adapter;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.BaseViewHolder;
import com.wyang.study.R;
import com.wyang.study.bean.Simple;

/**
 * Created by weiyang on 2019/6/19.
 */
public class SimpleAdapter extends BaseQuickAdapter<Simple, BaseViewHolder> {

    public SimpleAdapter() {
        super(R.layout.item_simple_recycler);
    }

    @Override
    protected void convert(BaseViewHolder helper, Simple item) {
        helper.setText(R.id.tv_title, item.getTitle());
        helper.setText(R.id.tv_description, item.getDescription());
    }
}

File: app/src/main/java/com/wyang/study/adapter/SimpleAdapter.kt
package com.wyang.study.adapter

import com.w6539android.base.ui.recycler.BaseDifferAdapter
import com.w6539android.base.ui.recycler.BaseViewHolder
import com.wyang.study.R
import com.wyang.study.bean.Simple

class SimpleAdapter : BaseDifferAdapter<Simple>(R.layout.item_simple_recycler) {
    override fun convert(holder: BaseViewHolder, item: Simple) {
        holder.setText(R.id.tv_title, item.title)
            .setText(R.id.tv_description, item.description)
    }
}